File: data/StarCoder2/readable_format_HPEC/cpp/file015.cpp
#include <iostream>
#include <omp.h>

int main() {
    int x = 2;

    #pragma omp task shared(x) mergeable
    {
        x = x + 1;
    }

    std::cout << "x = " << x << std::endl;

    return 0;
}

File: data/CodeLlama/readable_format_HPEC/fortran/file015.f90
program DRB130_mergeable_taskwait_orig_no
    use omp_lib
    implicit none

    integer :: x
    x = 2

    !$omp task shared(x) mergeable
    x = x+1
    !$omp end task

    print 100, x
    100 format ('x =',3i8)
end program
